test/unit/specs/core/network/injectSendBeaconRequest.spec.js

```diff
@@ -24,46 +24,60 @@ const guardForSendBeaconAvailability = spec => {
     : () => pending("No sendBeacon API available.");
 };
 
-describe("injectSendBeacon", () => {
+describe("injectSendBeaconRequest", () => {
   it(
-    "falls back to fetch if sendBeacon fails",
+    "falls back to sendFetchRequest if sendBeacon fails",
     guardForSendBeaconAvailability(() => {
       const navigator = {
         sendBeacon: jasmine.createSpy().and.returnValue(false)
       };
-      const fetchPromise = Promise.resolve();
-      const fetch = jasmine.createSpy().and.returnValue(fetchPromise);
+      const sendFetchRequestPromise = Promise.resolve();
+      const sendFetchRequest = jasmine
+        .createSpy()
+        .and.returnValue(sendFetchRequestPromise);
       const logger = {
         log: jasmine.createSpy()
       };
-      const sendBeacon = injectSendBeacon(navigator, fetch, logger);
+      const sendBeaconRequest = injectSendBeaconRequest({
+        navigator,
+        sendFetchRequest,
+        logger
+      });
       const body = { a: "b" };
-      const result = sendBeacon("https://example.com/endpoint", body);
+      const result = sendBeaconRequest("https://example.com/endpoint", body);
       expect(navigator.sendBeacon).toHaveBeenCalledWith(
         "https://example.com/endpoint",
         jasmine.any(Object)
       );
-      expect(fetch).toHaveBeenCalledWith("https://example.com/endpoint", body);
+      expect(sendFetchRequest).toHaveBeenCalledWith(
+        "https://example.com/endpoint",
+        body
+      );
       expect(logger.log).toHaveBeenCalledWith(
         jasmine.stringMatching("call has failed")
       );
-      expect(result).toBe(fetchPromise);
+      expect(result).toBe(sendFetchRequestPromise);
     })
   );
 
   it(
-    "does not fall back to fetch if sendBeacon succeeds",
+    "does not fall back to sendFetchRequest if sendBeacon succeeds",
     guardForSendBeaconAvailability(() => {
       const navigator = {
         sendBeacon: jasmine.createSpy().and.returnValue(true)
       };
       const body = { a: "b" };
-      const fetch = jasmine.createSpy();
-      const sendBeacon = injectSendBeacon(navigator, fetch);
-      // eslint-disable-next-line consistent-return
-      return sendBeacon("https://example.com/endpoint", body).then(() => {
-        expect(fetch).not.toHaveBeenCalled();
+      const sendFetchRequest = jasmine.createSpy();
+      const sendBeaconRequest = injectSendBeaconRequest({
+        navigator,
+        sendFetchRequest
       });
+      // eslint-disable-next-line consistent-return
+      return sendBeaconRequest("https://example.com/endpoint", body).then(
+        () => {
+          expect(sendFetchRequest).not.toHaveBeenCalled();
+        }
+      );
```

Comment:
Nothing really changed as far as logic goes in these tests. It's just renaming and sending arguments into `injectSendBeaconRequest ` as an object.